### PR TITLE
Challenge 6 - added POM for each page

### PR DESCRIPTION
**Pull request by milicazivanovic-work**
Please let me know if the code can be written better/clearer etc. Thanks!

### CODE REVIEW

File: cypress/e2e/cart.cy.js [hunk 1]
```diff
@@ -8,10 +12,10 @@ beforeEach(() => {
 
 describe("SauceDemo website test, continue on checkout without user info", () => {
   it("Opens checkout, tries to continue, expect errors", () => {
-    cy.get("#shopping_cart_container").click();
-    cy.get('[data-test="checkout"]').click();
-    cy.get('[data-test="continue"]').click();
-    cy.get('[data-test="error"]').should(
+    HomePage.shoppingCartBtn.click();
```
Comment (srdjadzogaz) [inline cypress/e2e/cart.cy.js:15]: Another good aproach is to have scenarios on per page. Scenarios or it blocks.

File: cypress/e2e/cart.cy.js [hunk 2]
```diff
@@ -20,48 +24,46 @@ describe("SauceDemo website test, continue on checkout without user info", () =>
 
 describe("SauceDemo website test, finish checkout", () => {
```
[the remark this answers is not in the document]
Comment (milicazivanovic-work) [reply cypress/e2e/cart.cy.js:25]: Thanks, I'll give it a go

**Review by srdjadzogaz — COMMENTED** [review]
Excelent. 
You have implemented POM correctly. 

In general test card.cy

**Review by srdjadzogaz — APPROVED** [review]

**Comment by milicazivanovic-work** [comment]
@srdjadzogaz I've tried to break the longest scenario into smaller chunks, not sure is this what you meant?

**Review by srdjadzogaz — APPROVED** [review]
> @srdjadzogaz I've tried to break the longest scenario into smaller chunks, not sure is this what you meant?  

Exactly, go ahead and merge.

**Pull request merged**